.github/workflows/on_merge_to_main.yml

```diff
@@ -183,6 +183,7 @@ jobs:
           exitOnceUploaded: true
         env:
           STORYBOOK_GITHUB_SHA: ${{ github.sha }}
+          NODE_OPTIONS: --max-old-space-size=6144
```

Comment:
Suggestion: worth a comment here to explain why we need this? Just so no one deletes it later